src/applications/mhv-secure-messaging/components/Draft/DeleteDraft.jsx

```diff
@@ -58,29 +63,46 @@ const DeleteDraft = props => {
       setIsModalVisible(false);
       dispatch(deleteDraft(draftId)).then(() => {
         if (draftsCount === 1) {
-          navigateToFolderByFolderId(
-            activeFolder ? activeFolder.folderId : DefaultFolders.DRAFTS.id,
-            history,
-          );
+          const { pathname } = location;
+          const defaultFolderId = activeFolder
+            ? activeFolder.folderId
+            : DefaultFolders.DRAFTS.id;
+
+          if (pathname.includes('/new-message')) {
+            navigateToFolderByFolderId(
+              activeFolder ? activeFolder.folderId : DefaultFolders.DRAFTS.id,
+              history,
+            );
+          }
+
+          if (pathname.includes('/reply')) {

```

Comment:
i think this can be addressed by using paths from constants the same way as code block below